package-lock.json

```diff
@@ -1961,6 +1974,7 @@
           "version": "1.0.2",
           "bundled": true,
           "dev": true,
+          "optional": true,
```

Comment:
Please dont submit the lock file unless you would have added some new dependencies